These files already installed by hipcc, which is a runtime dep
--- a/hipamd/CMakeLists.txt
+++ b/hipamd/CMakeLists.txt
@@ -401,16 +401,7 @@ if (NOT ${HIPCC_BIN_DIR} STREQUAL "")
       set(hipcc_bin ${hipcc_bin}.exe)
       set(hipconfig_bin ${hipconfig_bin}.exe)
     endif()
-    if(EXISTS ${hipcc_bin} AND EXISTS ${hipconfig_bin})
-      install(PROGRAMS ${hipcc_bin} DESTINATION bin)
-      install(PROGRAMS ${hipconfig_bin} DESTINATION bin)
-    endif()
   endif()
-  install(PROGRAMS ${HIPCC_BIN_DIR}/hipcc DESTINATION bin)
-  install(PROGRAMS ${HIPCC_BIN_DIR}/hipconfig DESTINATION bin)
-  install(PROGRAMS ${HIPCC_BIN_DIR}/hipcc.pl DESTINATION bin)
-  install(PROGRAMS ${HIPCC_BIN_DIR}/hipconfig.pl DESTINATION bin)
-  install(PROGRAMS ${HIPCC_BIN_DIR}/hipvars.pm DESTINATION bin)
   if(WIN32)
     install(PROGRAMS ${HIPCC_BIN_DIR}/hipcc.bat DESTINATION bin)
     install(PROGRAMS ${HIPCC_BIN_DIR}/hipconfig.bat DESTINATION bin)